extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java

```diff
@@ -66,10 +69,10 @@ public static SupplierEx<RestClientBuilder> elasticClientSupplier() {
 
     public static SupplierEx<RestClientBuilder> secureElasticClientSupplier() {
         ElasticsearchContainer container = elastic.get();
-        String containerIp = container.getContainerIpAddress();
+        String containerHost = container.getHost();
```

Comment:
Method `elasticClientSupplier()` uses `elastic.get().getHttpHostAddress()` and this is using `elastic.get().getHost()`. Is there any reason for having them different? Should not we unify it?

Reply:
+1, I unified those methods